index.js

```diff
@@ -17,24 +17,27 @@
     under the License.
 */
 
-var path = require('path');
 var fs = require('fs');
+var os = require('os');
+var path = require('path');
+
+var Q = require('q');
```

Comment:
I think the `Q.fcall(function () { })` could be replaced with `new Promise(function (resolve, reject) { })` pretty easily to remove the dependency on Q in favour of native promises

Reply:
See 8169cad